app/assets/javascripts/discourse/app/pre-initializers/sniff-capabilities.js

```diff
@@ -22,7 +22,7 @@ export default {
     caps.isAndroid = ua.includes("Android");
     caps.isWinphone = ua.includes("Windows Phone");
     caps.isOpera = !!window.opera || ua.includes(" OPR/");
-    caps.isFirefox = typeof InstallTrigger !== "undefined";
+    caps.isFirefox = "InstallTrigger" in window || ua.includes("Firefox");
```

Comment:
Should we flip these around so we hit the user agent first?

Reply:
We might not need the `InstallTrigger` at all, but for now I wanted to preserve the current behavior and added the `ua` check as a fallback for the future.

Reply:
We're only using this in one place in core (and zero places in plugins/themes), maybe we can already go for the `ua` check. Thoughts @xfalcox (since you worked just yesterday on the one place in core where we use `isFirefox`?